src/parsers/srdf.cpp

```diff
@@ -141,10 +141,75 @@ namespace se3
     }
     
 #endif // ifdef WITH_HPP_FCL
+    Eigen::VectorXd getRotorParamsFromSrdf(Model & model,
```

Comment:
Why returning Eigen::VectorXd while you are in fact returning some bool values.

Reply:
This has been corrected already